fe/fe-core/src/main/java/com/starrocks/qe/scheduler/DefaultWorkerProvider.java

```diff
@@ -287,17 +287,26 @@ private void reportWorkerNotFoundException(boolean chooseComputeNode) throws Non
 
     private String computeNodesToString() {
         StringBuilder out = new StringBuilder("compute node: ");
-        id2ComputeNode.forEach((backendID, backend) -> out.append(
-                String.format("[%s alive: %b inBlacklist: %b] ", backend.getHost(),
-                        backend.isAlive(), SimpleScheduler.isInBlocklist(backendID))));
+
+        id2ComputeNode.forEach((backendID, backend) -> {
+            if (!backend.isAlive() || SimpleScheduler.isInBlocklist(backendID)) {
+                out.append(
+                        String.format("[%s alive: %b inBlacklist: %b] ", backend.getHost(),
+                                backend.isAlive(), SimpleScheduler.isInBlocklist(backendID)));
+            }
+        });
         return out.toString();
     }
```

Comment:
I'm concerned that this function will be used in different scenarios. After this change, the function doesn't correspond to the name of this function.

Reply:
> I'm concerned that this function will be used in different scenarios. After this change, the function doesn't correspond to the name of this function.

add one argument to determine whether to print alive backend